modular_skyrat/master_files/code/modules/events/spacevine.dm

```diff
@@ -477,6 +477,17 @@
 	var/turf/vine_object_turf = get_turf(vine_object)
 	vine_object_turf.atmos_spawn_air("miasma=100;TEMP=[T20C]")
 
+// Generates plasma on growth
+/datum/spacevine_mutation/plasma_generating
+	name = "Plasma producing"
+	hue = "#470566"
+	severity = 5
+	quality = NEGATIVE
+
+/datum/spacevine_mutation/plasma_generating/on_grow(obj/structure/spacevine/vine_object)
+	var/turf/vine_object_turf = get_turf(vine_object)
+	vine_object_turf.atmos_spawn_air("plasma=100;TEMP=[T20C]")
```

Comment:
That looks like a recipe for disaster, I'm not sure that's something we want out of vines. I feel like this would make them *too* round-ending, which really isn't what we want from something that wasn't designed to be round-ending to begin with.